src/Nethermind/Nethermind.TxPool/CompetingTransactionEqualityComparer.cs

```diff
@@ -20,13 +20,18 @@
 
 namespace Nethermind.TxPool
 {
-    public class PendingTransactionComparer : IEqualityComparer<Transaction>
+    /// <summary>
+    /// Comparer to check if two pending transactions compete with each other
+    /// </summary>
+    public class CompetingTransactionEqualityComparer : IEqualityComparer<Transaction>
```

Comment:
Some more comments would be desired or some renaming. It took me a long time to find info / understand what it does.